.github/workflows/update-contributors.yml

```diff
@@ -15,4 +15,4 @@ jobs:
       - name: Update contributors in README
         uses: akhilmhdh/contributors-readme-action@v2.3
         env:
-          GITHUB_TOKEN: ${{ secrets.GH_TOKEN }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Use GH_TOKEN to reference the not GITHUB_TOKEN.
GH_TOKEN is the configured one for this project